### ISSUE TITLE
Remote cloned presentation?

### ISSUE THREAD

**Issue by RobGibbens** [gnab/remark#203]
Is it possible to clone a presentation from a remote computer (to see presenter notes)?

**Comment by gnab** [comment]
Sorry for the late reply. Currently this is not a supported feature of remark.

**Comment by mcowger** [comment]
Is it possible to do on the same computer?  If not, what is the value of presenter mode?

**Comment by gnab** [comment]
@mcowger Yes, by pressing __C__ you get a separate window that will follow the navigation in the original that can be shows on a projector, and then you can turn on presenter notes in the original one and keep that on the presenter's display.

**Comment by omnibs** [comment]
I did this for myself, but thought others might find use:

https://github.com/omnibs/remarksync

Synchronizes remark presentations across devices on the internet.
